### DISCUSSION TITLE
RPi 4 - Minimum memory requirement?

### DISCUSSION THREAD

**Discussion by DaveBKC** [mcguirepr89/BirdNET-Pi#368] [Q&A] [answered]
One thing that's not in the requirements section is the recommended minimum amount of memory. From that, I assume that most people are using the 2gb model. 

Question: Is there any benefit to running BirdNET-Pi on a 4gb or even 8gb device?

**Comment by DD4WH** [comment, marked as answer]
Dave,
I have tried BirdNET-Pi on all models of Raspberry Pi 4B --> 1GB, 2GB, 4GB and 8GB.
I have not tested systematically, but it seems to run on all models in the same way.

**Comment by ehpersonal38** [comment]
My RPi4B (used exclusively for BirdNET-Pi) averages about 900MB ram, with peaks up to 1.2GB, for what it's worth.

**Comment by DaveBKC** [comment]
Thanks folks! Both answers answer the question, so I had to do an eeny, meeny, miny, moe to pick one.

**Comment by srchild** [comment]
Maybe the [proposed change to making use of Ramdisk](https://github.com/mcguirepr89/BirdNET-Pi/discussions/247) will mean that extra memory may be more relevant in future than at present?

**Reply by DaveBKC** [reply]
Could well be.  With the price of SSD's these days, I ended up using a 240G Kingston in place of an SD card to avoid SD-related issues.
